layout/layout.h

```diff
@@ -175,9 +175,9 @@ enum {
 	enum { M0_DEFAULT_LAYOUT_ID = 1 };
 #elif defined CONFIG_AARCH64
 /**
- * Layout id defines the object unit size which cannot be less than the page size:
- * 1 - 4KB, 2 - 8KB, 3 - 16KB and so on. The page size on RHEL aarch64 platform is
- * 64KB, hence the default layout id should be 5 (64KB).
+ * Layout id defines the object unit size which cannot be less than 
+ * the page size:1 - 4KB, 2 - 8KB, 3 - 16KB and so on. The page size on 
+ * RHEL aarch64 platform is 64KB, hence the default layout id should be 5 (64KB).
```

Comment:
This line is still longer than 80 chars. Why don't take the variant which I suggested?

Reply:
Fixed

Reply:
fixed